mlir-tensorrt/compiler/lib/Conversion/TensorRTRuntimeToExecutor/TensorRTRuntimeToExecutor.cpp

```diff
@@ -375,6 +375,13 @@ struct ConvertEnqueueAllocToCall
     for (auto [idx, result] : llvm::enumerate(op.getResults())) {
       MemRefType memrefType = cast<MemRefType>(result.getType());
       unsigned rank = memrefType.getRank();
+
+      Value rankOffset = b.create<executor::GetOffsetOp>(
```

Comment:
Just do `outputDescOffset++`. It's not necessary to create extra SSA values that will just get removed later.